pkg/stanza/operator/parser/syslog/config.go

```diff
@@ -55,6 +55,7 @@ type BaseConfig struct {
 	EnableOctetCounting          bool    `mapstructure:"enable_octet_counting,omitempty"`
 	AllowSkipPriHeader           bool    `mapstructure:"allow_skip_pri_header,omitempty"`
 	NonTransparentFramingTrailer *string `mapstructure:"non_transparent_framing_trailer,omitempty"`
+	MaxLogSize                   int     `mapstructure:"max_log_size,omitempty"`
```

Comment:
Perhaps for the sake of clarity it would be better to call this `max_octets`? This would more clearly relate the value to octet counting and also would differentiate it from tcp's similar setting.

Reply:
good suggestion, I will adapt the name of this option